modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -313,10 +313,18 @@ export default class I3SConverter {
     if (sourceTile.id) {
       console.log(`[analyze]: ${sourceTile.id}`); // eslint-disable-line
     }
-    const tileContent = await loadTile3DContent(this.sourceTileset, sourceTile, {
-      ...this.loadOptions,
-      '3d-tiles': {...this.loadOptions['3d-tiles'], loadGLTF: false}
-    });
+
+    let tileContent: Tiles3DTileContent | null = null;
+    try {
+      tileContent = await loadTile3DContent(this.sourceTileset, sourceTile, {
+        ...this.loadOptions,
+        '3d-tiles': {...this.loadOptions['3d-tiles'], loadGLTF: false}
+      });
+    } catch (error) {
+      console.log(
+        `[warning]: Failed to load ${sourceTile.contentUrl}. This tile will be converted with empty content`
```

Comment:
Converting empty content sounds a little strange, Can we slightly simplify this message, e.g.? 
```suggestion
        `[warning]: Failed to load ${sourceTile.contentUrl}. An I3S tile with empty content will be added to the output tileset..`
```
Also should we offer an option not to include empty tiles? What is the advantage of doing so?